packages/docs/pages/reference/elements/sage.mdx

```diff
@@ -8,12 +8,82 @@ import {
 
 # `<sage>`
 
-Short description of what the `<sage>` tag does.
+The `<sage>` tag allows the author to incorporate the computational resources of
+the open source library `www.sagemath.org` into PreTeXt.
 
+Quoting from the PreTeXt Guide, a "typical use of the `<sage>` tag is to 
```

Comment:
Let's paraphrase rather than quote. We can link to the appropriate section of the guide here, too.